src/score_hv/harvesters/innov_netcdf.py

```diff
@@ -358,11 +358,11 @@ def get_data(self):
         harvested_data = []
         for metric in metrics:
             ncfile = netCDF4.Dataset(metric.filename)
-            try:
-                ev_units = self.config.get_elevation_units()
+            if True:#try:
```

Comment:
This looks funky. seems like you didnt mean to remove the try block.

Reply:
This long try block makes debugging difficult by obscuring a source of error within. Furthermore, it would be helpful to use the except clause to catch specific exceptions. For example, here is an example error message generated from a first try of this code: 
`ValueError: problem parsing netcdf file /Users/aschneider/projects/ufs-rnr/src/score-hv/tests/data/innov_stats.temperature.2015120106.nc - err: None`

This error message was not helpful. In this case, simply removing the try/except block narrowed down the issue and allowed for easier debugging using Python's standard error messages.

Because the try block is so long and the except clause is nonspecific, I do not understand what errors are trying to be caught here, only to be reraised with an obscure custom message.

I believe the PEP supports my preference for shorter try blocks with more specific exception handling:
https://peps.python.org/pep-0008/#programming-recommendations